CONTRIBUTING.md

```diff
@@ -189,6 +189,46 @@ Take a look at the sections that follow to pick a route (or five) that appeal to
 
 If you ever find yourself at a loss for what to do, or in need of mentorship or advice on how to contribute to Bevy, feel free to ask in [Discord] and one of our more experienced community members will be happy to help.
 
+### Join a working group
+
+Active initiatives in Bevy are organized into temporary working groups: choosing one of those and asking how to help can be a fantastic way to get up to speed and be immediately useful.
+
+Working groups are public, open-membership groups that work together to tackle a broad-but-scoped initiative.
+The work that they do is coordinated in a forum-channel on [Discord](https://discord.gg/bevy), although they also create issues and may use project boards for tangible work that needs to be done.
+
+There are no special requirements to be a member, and no formal membership list.
+Working groups are *spaces*, not clubs.
```

Comment:
This sounds pretty good for those wanting to join a group, but could be an undesirable aspect of creating one.

When someone asks to join a group, the group founders or current members may wonder:

- "Can this newcomer be trusted?"
- "Are they truly as dedicated as us? Will they put in the same level and quality of work?"
- "Will we have to slow down momentum to catch them up to speed or wait for their PR to meet the group's criteria?"

These are valid concerns such groups might have and may be further fueled by however much effort it took to get the two other members and maintainer approval needed to even get the initiative off the ground.

Of course, Bevy is an open-source project: you're supposed to work with people. 

I don't think group membership should be closed. But this might be something we want to more explicitly address in these new sections. There should be an expectation by those starting a working group that others can join/leave at any time. Part of making a public group should be to welcome in those interested in joining and respectfully working around people needing to step away. Otherwise, you and your team are probably better off working on it outside of the Working Group system.

Reply:
I think we should avoid the idea of "founders" or anyone else having ownership over a group. In fact, excluding someone from a work group a should be viewed as a Code of Conduct violation. It might be good to highlight this more.

As I see it: A work group is a narrowly-scoped discussion with the end goal of designing and implementing a feature. Everyone is welcome to participate in the discussion and contribute to the work. If you do choose to participate, leaving a review on the resulting PRs should be *strongly encouraged*.

Reply:
> I think we should avoid the idea of "founders" or anyone else having ownership over a group.

Yes, good point! I should have used a different term. That is definitely not a sentiment we want to promote. I think what I'm getting at is there might be a "sense of ownership" amongst a group's current members that could make it difficult to share the initiative with newcomers.

So I think we should make it really clear that the group should _expect_ others to join/leave.

> And excluding someone from a work group a should be viewed as a Code of Conduct violation, IMO. It might be good to highlight this more.

Yeah, it goes against the spirit of collaboration. If you want to take advantage of this new system, you have to be willing to work with others. Otherwise, you'll have to go about pushing for your changes the ol' fashioned way.

Reply:
That's a really good point. I absolutely agree.